doc/proxy-clients.rst

```diff
@@ -24,6 +24,7 @@ Client        Purge   Refresh Ban     Tagging Clear
 ============= ======= ======= ======= ======= =======
 Varnish       ✓       ✓       ✓       ✓
 NGINX         ✓       ✓
+LiteSpeed     ✓               ✓       ✓       ✓
```

Comment:
the client does not implement the BanCapable interface - can we support BAN or not?

Reply:
I'll update the docs. Both Purge and Ban could be implemented by `PURGE` requests as for example in Varnish. Unfortunately this is not possible for tags which is why I've implemented this file-generation-thingy in the first place.